recipes/_configure_server.rb

```diff
@@ -75,6 +75,15 @@
 
   server_stanzas['clustering'] = settings
   server_stanzas['clustering']['mode'] = 'master'
+
+  indexer_discovery_settings = ((bag['indexer_discovery_settings'] && bag['indexer_discovery_settings']['master_configs']) || {}).reject do |k, _|
+    k.start_with?('_cerner_splunk')
+  end
+  pass = (bag['indexer_discovery_settings'] && bag['indexer_discovery_settings']['pass4SymmKey']) || nil
+
+  server_stanzas['indexer_discovery'] = indexer_discovery_settings if bag['indexer_discovery_settings']
+  server_stanzas['indexer_discovery']['pass4SymmKey'] = CernerSplunk::ConfTemplate.compose encrypt_password, CernerSplunk::ConfTemplate::Value.constant(value: pass) if pass
+
 when :cluster_slave
```

Comment:
For Cluster Slave, I think there are some additional configuration options we may want to support... namely `register_forwarder_address` under the `[clustering]` stanza... there are other interesting settings too like `register_replication_address` and `register_search_address` which would likely be set via procs if necessary so would need that sort of discovery and injection

Reply:
Talked to Charlie about this yesterday. 
For now, we will move forward without configuring this attribute. Maybe once we are at a point where we can deploy this in AWS and test it, we will consider adding these attributes if required. Charlie suggested using the Instance Metadata (http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html) to find the instance public ip if there is one.